code/obj/machinery/door/airlock.dm

```diff
@@ -578,6 +578,9 @@ var/global/list/cycling_airlocks = list()
 				return
 	..()
 /obj/machinery/door/airlock/bumpopen(atom/movable/AM)
+	if (src.isWireCut(AIRLOCK_WIRE_IDSCAN))
+		src.play_deny() // intentional nerf: idwire-cut doors won't respond to bumps. incentivizes people to fix it and makes them aware it is access hacked.
+		return
```

Comment:
feels a bit odd to handle this here with an early return that doesn't call parent, because this doesn't add fingerprints for example, and looks like it wouldn't respect the cooldown on the deny sfx the parent proc sets. Is there a better way to handle this?

Reply:
honestly, not really. i can't move the logic to door parent, this is a nerf exclusive to airlocks. the only other ways i can think of doing is either adding a new `should_open` var to the parent's proc, and changing values on `src` so the conditions in the parent's proc fails, both solutions very janky to me.

Reply:
Mmm
Janky turtles all the way down, I guess